keyboards/orange75/readme.md

```diff
@@ -0,0 +1,16 @@
+# Orange75
+
+![Orange75](https://i.imgur.com/eAfiwwM.jpg)
+
+A 75% made by Fox Lab.
+
+Keyboard Maintainer: [captsis](https://github.com/captsis), [fox-lab](https://github.com/fox-lab)   
+Hardware Supported: Orange75 QMK  
```

Comment:
A note here about this firmware supporting only the 32u4 version of the Orange75 would be good. Perhaps 

```
Hardware Supported: Orange75 powered by the ATmega32U4  
```

I bring this up only because QMK can power the 32A Bootmapper Client version of the PCB as well, but that one would need different source code from the 32U4 version. The [Rama M65-A](https://github.com/qmk/qmk_firmware/tree/master/keyboards/jc65/) is an example of this.